src/Authentication/Authentication/custom/common/Permissions.ps1

```diff
@@ -1,36 +1,49 @@
 # ------------------------------------------------------------------------------
 #  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 # ------------------------------------------------------------------------------
-
-
-$permissionsPulledFromMgGraphRequest = $null
+Update-FormatData -PrependPath $PSScriptRoot\..\..\Microsoft.Graph.Authentication.format.ps1xml
```

Comment:
This should not be done here -- let's do this in the .psd1 file -- just add this path to `formatsToProcess`.